scripts/processors/cams_processor.py

```
@@ -0,0 +1,343 @@
+        
+        Args:
+            start_date: Start date for processing
+            end_date: End date for processing
+            output_file: Output CSV file path
+            
+        Returns:
+            Processed DataFrame
+        """
+        # Set default dates if not provided
+        if not start_date:
+            start_date = datetime.now() - timedelta(days=30)
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)